### PR TITLE
TOS-97

### PR DESCRIPTION
**Pull request by Raksha-testsigma**

### CODE REVIEW

**Review by rajesh-testsigma — COMMENTED** [review]
Looks good. U need to add the SQL Changes to Migration file as well.

File: server/src/main/resources/db/bootstrap/V208__bootstrap_action_examples.sql [hunk 1]
```diff
@@ -1,57 +1,96 @@
...
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `natural_text_action_examples`
+--
+
+DROP TABLE IF EXISTS `natural_text_action_examples`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `natural_text_action_examples` (
```
Comment (tarun-testsigma) [inline server/src/main/resources/db/bootstrap/V208__bootstrap_action_examples.sql:25]: There is already a create table command in V35_actions_examples.sql . Why do we need this ?

Comment (Raksha-testsigma) [reply server/src/main/resources/db/bootstrap/V208__bootstrap_action_examples.sql:25]: So this can be removed?

Comment (tarun-testsigma) [reply server/src/main/resources/db/bootstrap/V208__bootstrap_action_examples.sql:25]: Remove this and test

**Pull request closed without merging**